packages/lab/src/Wizard/Wizard.js

```diff
@@ -0,0 +1,186 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/core";
+import WizardContext from "./WizardContext";
+import WizardContainer from "./WizardContainer";
+import WizardTitle from "./WizardTitle";
+import WizardContent from "./WizardContent";
+import WizardActions from "./WizardActions";
+
+import styles from "./styles";
+
+/**
+ * Wizard description/documentation paragraph
+ */
```

Comment:
This is a leftover comment either fill the description or erase it

Reply:
removed the comment and changed some Portuguese labels to English.